src/Store.js

```diff
@@ -78,6 +78,37 @@ class Store {
     return transaction(() => data.map((obj) => this.addModel(type, obj)))
   }
 
+  /**
+   * Saves a collection of records via a bulk-supported JSONApi
+   * endpoint. All records need to be of the same type.
+   *
+   * @method bulkSave
+   * @param {String} type
+   * @param {Array} records
+   */
+  bulkSave = async (type, records, options = {}) => {
+    const { queryParams } = options
+
+    // get url for record type
+    const url = this.fetchUrl(type, queryParams, null)
+
+    // convert records to an appropriate jsonapi attribute/relationship format
+    const recordAttributes = records.map((record) => record.jsonapi())
+
+    // build a data payload
+    const body = JSON.stringify({ data: recordAttributes })
+
+    // send request
+    const response = this.fetch(url, {
+      headers: { ...this.defaultFetchOptions.headers, 'Content-Type': 'application/vnd.api+json; ext="bulk"' },
+      method: 'POST',
+      body
+    })
+
+    // update records based on response
+    await this.updateRecords(response, records)
```

Comment:
Let's return this promise so the caller can `await` it.
```suggestion
    return this.updateRecords(response, records)
```